server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/rule/MySQLAffectedRowsExceedLimit.java

```diff
@@ -76,17 +81,42 @@ public SqlCheckRuleType getType() {
     @Override
     public List<CheckViolation> check(@NonNull Statement statement, @NonNull SqlCheckContext context) {
 
+        long affectedRows = getAffectedRows(statement);
```

Comment:
we may extract a method to get affect row count

Reply:
yes, that's what I did.